docs/kubernetes/KubernetesPolicy.md

```diff
@@ -0,0 +1,62 @@
+# Programming Calico Policy in Kubernetes
+The [Calico-Kubernetes plugin](https://github.com/projectcalico/calico-docker/blob/master/docs/kubernetes/KubernetesIntegration.md) will allow you to program Networking policy directly from your Kubernetes pod config. 
+
+
+### The Basics
```

Comment:
This should be "Declaring Policy", and explain how a user can declare policy.

"The Calico Networking plugin will look for a `policy` key" --- I think this is too implementation oriented.  The goal here is to allow end-users to easily declare policy, so we shouldn't need to specify anything implementation specific. (For example, why should they care if annotations are passed over the networking plugin API, or if the plugin queries for the annotations?)